packages/dd-trace/src/ci-visibility/exporters/agentless/di-logs-writer.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+const request = require('../../../exporters/common/request')
+const log = require('../../../log')
+const { safeJSONStringify } = require('../../../exporters/common/util')
+const { JSONEncoder } = require('../../encode/json-encoder')
+
+const BaseWriter = require('../../../exporters/common/writer')
+
+// Writer used by the integration between Dynamic Instrumentation and Test Visibility
+// It is used to encode and send logs to both the logs intake directly and the
+// `/debugger/v1/input` endpoint in the agent, which is a proxy to the logs intake.
+class DynamicInstrumentationLogsWriter extends BaseWriter {
+  constructor ({ url, isAgentProxy = false }) {
+    super(...arguments)
+    this._url = url
+    this._encoder = new JSONEncoder()
+    this._isAgentProxy = isAgentProxy
+  }
+
+  _sendPayload (data, _, done) {
+    const options = {
+      path: '/api/v2/logs',
+      method: 'POST',
+      headers: {
+        'dd-api-key': process.env.DATADOG_API_KEY || process.env.DD_API_KEY,
+        'Content-Type': 'application/json'
+      },
+      timeout: 15000, // TODO: what's a good value for timeout for the logs intake?
```

Comment:
Perhaps it makes sense to make it configurable with a default of 15s?

Reply:
yeah that makes sense